abce/__init__.py

```diff
@@ -336,7 +309,7 @@ def declare_service(self, human_or_other_resource,
         """
         self.declare_round_endowment(
             human_or_other_resource, units, service)
-        self.declare_perishable(service)
```

Comment:
I would like spin of declare_expirenig into a plugin, as it is used only in very special circumstances. For I'd say 80% uses is serciveces, where perisable sufficies 16% perishable is sufficient and in only the remaining 4% expireing is used. Expireing is complicated and slow.